### ISSUE TITLE
npm link shared mojits failed on 0.7.2

### ISSUE THREAD

**Issue by ftliou** [YahooArchive/mojito#1219]
Please refer to:
http://developer.yahoo.com/forum/Yahoo-Mojito/npm-link-shared-mojits-failed-on/1376445752796-c287e870-fd51-473e-ac9a-557335d087a3

In mojito/lib/app/autoload/store.server.js

```
       if (!this._libs.fs.lstatSync(dir).isDirectory()) {
            return;
        }
```

This block of code stops npm linked shared mojits from being loaded.

**Comment by caridy** [comment]
regression introduced by fix for issue #1017 /cc @lzhan @isao

**Comment by caridy** [comment]
@lzhan can you validate and close this?
